src/lib/mock-backend.js

```diff
@@ -1124,7 +1127,11 @@ var owner = {
         large: "https://randomuser.me/api/portraits/women/82.jpg",
         medium: "https://randomuser.me/api/portraits/med/women/82.jpg",
         thumbnail: "https://randomuser.me/api/portraits/thumb/women/82.jpg"
-    }
+    },
+    users: [new User('Erkek', 'Armağan Amcalar', 'https://randomuser.me/api/portraits/men/67.jpg'),
```

Comment:
Why is the gender name in Turkish? What happens if someone makes a typo when writing this string (`'erkek'` instead of `'Erkek'` for example)? How about using an enum here?